### ISSUE TITLE
Audio functions hang outside of notebook

### ISSUE THREAD

**Issue by balintlaczko** [fourMs/MGT-python#191]
While investigating #190, found this. On Windows, everything works as expected - both in the notebook, and in the terminal (via IPython or script call), on Ubuntu everything works as expected __in the notebook__. However, in the Terminal (using IPython or via script call) any audio function will cause the Terminal to hang forever (without any CPU use) which is quite odd.

**Comment by balintlaczko** [comment]
It seems to be some kind of conflict between librosa and matplotlib. (More details soon.)

**Comment by balintlaczko** [comment]
Minimal code to reproduce:

```
import matplotlib.pyplot as plt
import matplotlib
import librosa
import librosa.display

y, sr = librosa.load('dance.avi', sr=None)
print('Loaded file.')

fig, ax = plt.subplots(figsize=(12, 6), dpi=300)
print('Made plots.')
```

The progam will print 'Loaded file.' and then it halts forever (even KeyboardInterrupt won't work). If I am doing _either_ the `librosa.load` _or_ the `plt.subplots` everything is fine. If I do `plt.subplots` and after that `librosa.load` it will also work. But if I first load the file and _then_ make the subplots (like above) I get into the deadlock.

**Comment by balintlaczko** [comment]
Found reference: https://stackoverflow.com/questions/46806318/librosa-stops-matplotlib-from-working

**Comment by balintlaczko** [comment]
Somewhat intuitively found a fix, which also seem to solve #190 at the same time. It is the opposite of clever, but it works. In _audio.py_ after all imports:
```
plt.plot()
plt.close()
```
This somehow fixes everything. Tested in notebook/terminal on windows/ubuntu.
